src/components/text-input/TextInput.stories.mdx

```diff
@@ -188,18 +188,18 @@ export const ControlledTextInput = (args) => {
 
 export const noTrailingIconArgs = {
     ...defaultArgs,
-    trailingIcon: 'none',
+    trailingIcon: '',
```

Comment:
nit: an empty string is not a valid option as `null` would be for this prop, but since it serves the purpose this is fine for now!